src/request.ts

```diff
@@ -18,37 +18,44 @@ export const request = async <T = Response>(
     retries: requestSettings.retries,
   }
 ): Promise<T> => {
-  const { userId, integrator, widgetVersion } =
+  const { userId, integrator, widgetVersion, apiKey } =
     ConfigService.getInstance().getConfig()
 
   options.retries = options.retries ?? requestSettings.retries
   try {
     if (!options.skipTrackingHeaders) {
+      if (apiKey) {
+        options.headers = {
+          ...options?.headers,
+          'x-lifi-api-Key': apiKey,
```

Comment:
Should it be `x-lifi-api-key`? 🙂

Reply:
![image](https://github.com/lifinance/sdk/assets/8833906/f58e14c8-53e0-4213-a762-ebe8ff0df7ab)

Reply:
What a shitty PR. I am ashamed of myself